examples/tomee-jms-portability/README_pt.adoc

```diff
@@ -0,0 +1,90 @@
+:index-group: JMS
+:jbake-type: page
+:jbake-status: status=published
+# Portabilidade entre ActiveMQ e IBM MQ
+
+Este aplicativo demonstra a capacidade de obter portabilidade/interoperabilidade entre o ActiveMQ e o IBM MQ para aplicativos baseados no tipo MDB e conexão JMS baseada em Java. Abrange padrões e aprendizados conforme listado abaixo.
+
+1. Como executar vários contêineres mdb na mesma JVM;
+2. Várias maneiras de configurar IBM Filas/Tópicos vs AMQ Filas/Tópicos;
+3. Configuração do MDB para IBM MQ vs AMQ e Anotações vs Configuração;
+4. Declaração estática vs dinâmica/substituição de filas/tópicos;
+5. Uso e importância de RA (Adaptadores de Recursos) e Fábricas de Conexão;
+6. Amostras de serviço da Web para gravar dados em IBM Queues e AMQ Queues usando injeção de recursos (fábricas de conexão e filas/tópicos);
+7. Referência de JNDI para recursos (consulte configuração de WMQReadBean em ejb-jar.xml);
+
+
+## Executando este aplicativo
+### Ambiente
+O aplicativo pode ser importado na IDE Eclipse como um projeto maven e é testado usando apache-tomee-plume-7.0.2 e apache-activemq-5.14.3 (executando como standalone no localhost, IBM MQ 8.xx (detalhes das conexões ocultos para evitar mau uso, aqueles que desejam testar podem entrar em contato com a equipe de middleware para obter acesso à instância gerenciada do IBM MQ e ao AppWatch)).
+
+### Bibliotecas Adicionais
+Para conectar-se ao IBM MQ, o tomee requer bibliotecas adicionais contendo implementação para Adaptador de Recurso JCA, fábricas de conexão do IBM MQ, recursos de fila/tópico, ActivationConfigs, etc. Essas bibliotecas vêm com assinatura IBM e todas elas não estão disponíveis no maven central, abaixo é a lista de todos os arquivos jar específicos da IBM.
+
+com.ibm.mq.connector.jar  
+com.ibm.mq.jar      
+com.ibm.mq.pcf.jar   
+com.ibm.mq.headers.jar    
+com.ibm.mq.jmqi.jar   
+com.ibm.mqjms.jar
+providerutil.jar
+
+com.ibm.mq.connector.jar pode ser extraido de wmq.jmsra.rar que está disponível com a instalação do IBM MQ (não disponível na internet).
+
+
+### tomee.xml
+Este arquivo (localizado em tomee_home/conf é deixado em branco, pois gostaríamos que nosso aplicativo gerenciasse todos os recursos. Qualquer recurso declarado aqui estaria disponível para todos os aplicativos implantados no servidor.
+(Arquivo não incluido neste repositório git)
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<tomee>
+</tomee>
+----
+
+
+### catalina.properties
+Este arquivo pode ser usado para fornecer convinentemente argumentos java -D como uma entrada para tomee. No entanto, no mundo real, esses parâmetros seriam configurados usando variáveis de ambiente para evitar a adição de propriedades específicas do aplicativo no tomee conf (Arquivo não incluído neste repositório git).
```

Comment:
change "convinentemente" to "convinentemente"

Reply:
for me both are the same

Reply:
"convenientemente"

Reply:
Done, thanks @Daniel-Dos